File: src/main/java/ru/raiffeisen/custody/example/spingdata/dto/javaDto/ComponentDto.java
package ru.raiffeisen.custody.example.spingdata.dto.javaDto;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class ComponentDto {
    private final Long id;
    private final String name;
    private final String unit;
    private final CategoryDto category;
    private final String code;
    private final BigDecimal stock;
}

File: src/main/java/ru/raiffeisen/custody/example/spingdata/dto/ComponentDto.kt
package ru.raiffeisen.custody.example.spingdata.dto

import com.example.techcardservice.dto.CategoryDto
import lombok.Setter
import java.math.BigDecimal

@Setter
data class ComponentDto(
    val id: Long?,
    val name: String,
    val unit: String,
    val category: CategoryDto,
    val code: String,
    var stock: BigDecimal = BigDecimal(75)
)
